jstests/replsets/all_commands_downgrading_to_upgraded.js

```diff
@@ -261,6 +261,7 @@ const allCommands = {
         isAdminCommand: true,
         isShardSvrOnly: true,
     },
+    auditGetOptions: {skip: "already tested by 'audit' tests"},
```

Comment:
"Already tested ..." message is misleading here because all commands are tested in more than one place.
The goal of this test is to check that commands work in the downgrading FCV state.
We actually can execute 'auditGetOptions' command here. Just replace this line with:
```
auditGetOptions: {
    command: {auditGetOptions: 1},
},
```

Reply:
Added execution